airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -53,6 +56,16 @@
 PARAMS = {"dataRange": "LAST_14_DAYS", "timezoneCode": "America/New_York"}
 # [END howto_display_video_env_variables]
 
+# download_line_items variables
+OBJECT_NAME = "files/report.csv"
+GZIP = False
```

Comment:
Imho this GZIP can be inlined in operator invocation

Reply:
Sure, right.